dask/base.py

```diff
@@ -717,13 +717,13 @@ def normalize_function(func):
 
 
 def _normalize_function(func):
-    if isinstance(func, curry):
+    if isinstance(func, curry) and hasattr(func, "_partial"):
```

Comment:
Why was this change needed?

Reply:
Because cytoolz.curry and toolz.curry are not interchangeable. They have different attribute interfaces.

Reply:
Looks like I mis-spoke. Looking at it further with fresher eyes.

Reply:
Looks like we can remove that if statement entirely!